app/stacks/applications-service/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "ni_frontend_url" {

```

Comment:
Suggest making this variable a map that includes all frontend urls. Since we will have multiple in the appeals service. Can then combine them all in to one big map in the frontdoor stack (using merge()). That will allow us to dynamically add all frontend urls frontdoor without having a variable for each.

Reply:
Dynamic frontend endpoints, backends, and routing rules have now been configured and successfully deployed to test.